wolvic/browser/vr/wvr_manager.cc

```diff
@@ -499,6 +490,27 @@ WvrManager::GetInputSourceState() {
   return input_sources;
 }
 
+void WvrManager::DrawFrameSubmitNow() {
+  device::WebXrFrame* processing_frame = webxr_.GetProcessingFrame();
+  if (!SubmitFrameInternal(processing_frame->index))
+    return;
```

Comment:
We can pass the processing_frame as an argument and avoid calling GetProcessingFrame twice

Actually it's only used to call SubmitFrameInternal. To me this is an indication that these three lines should be on the caller and not here.

Reply:
done.